colocated-broccoli-plugin.js

```diff
@@ -14,6 +15,21 @@ module.exports = class ColocatedTemplateProcessor extends Plugin {
     this.options = options;
   }
 
+  _transformExistingComponent(filePath, jsContents, templateContents) {
+    if (!jsContents.includes('export default')) {
+      let message = `\`${filePath}\` does not contain a \`default export\`. Did you forget to export the component class?`;
+      return `${jsContents}\nthrow new Error(${JSON.stringify(message)});`;
+    }
```

Comment:
We can probably move this check/rewrite to the babel plugin now, for fewer false positives.

Reply:
Good idea!